hoomd/simulation.py

```diff
@@ -103,8 +104,8 @@ def create_state_from_snapshot(self, snapshot):
         self._state = State(self, snapshot)
 
         step = 0
-        if self.timestep is not None:
-            step = self.timestep
+        if self._timestep is not None:
```

Comment:
Fix this in the `timestep` property, as that will solve the problem for `snapshot`, `gsd`, and all future `create_state_from_*` methods.

Reply:
Fixed 7f0d69ad9